package.json

```diff
@@ -61,7 +61,7 @@
     "@opengovsg/angular-legacy-sortablejs-maintained": "^1.0.0",
     "@opengovsg/angular-recaptcha-fallback": "^5.0.0",
     "@opengovsg/formsg-sdk": "^0.8.4-beta.0",
-    "@opengovsg/myinfo-gov-client": "=3.0.0",
+    "@opengovsg/myinfo-gov-client": "^4.0.0-beta.0",
```

Comment:
should we fix the version until SDK is stable and makes it into the general release?